qboost.py

```diff
@@ -284,6 +284,28 @@ def __init__(self, X, y, lam, weak_clf_scale=None, drop_unused=True):
         super().__init__(weak_classifiers, weights, weak_clf_scale)
         self.fit_offset(X)
 
+        # Save candidates so we can provide a baseline accuracy report.
+        self._wclf_candidates = wclf_candidates
+
+    def report_baseline(self, X, y):
+        """Report accuracy of weak classifiers.
+
+        This provides context for interpreting the performance of the boosted
+        classifier.
+        """
+
+        from tabulate import tabulate
+
+        scores = []
+        for clf in self._wclf_candidates:
+            scores.append(accuracy_score(y, clf.predict(X)))
+        scores = np.array(scores)
```

Comment:
Minor suggestion for simplification:
```suggestion
        scores = np.array([accuracy_score(y, clf.predict(X))
                           for clf in self._wclf_candidates])
```

Reply:
Love it!  Will do.